docs/core/runtime-config/garbage-collector.md

```diff
@@ -771,3 +771,16 @@ Example *app.config* file:
 
 > [!TIP]
 > Experiment with different numbers to see which value works best for you. Start with a value between 5 and 7.
+
+## Dynamic adaptation to application sizes (DATAS)
+
+- Configures the garbage collector to use DATAS. DATAS adapts to application memory requirements, meaning the app heap size should be roughly proportional to the long-lived data size.
+- Enabled by default starting in .NET 9.
+
+|                          | Setting name               | Values    | Version introduced |
+|--------------------------|----------------------------|-----------|--------------------|
+| **runtimeconfig.json**   | `System.GC.DynamicAdaptationMode` | `1` - enabled<br/> `0` - disabled | .NET 8 |
```

Comment:
This order doesn't strike me as correct. Given the current order, I'd go with: 2, 3, 1. runtimeconfig and MSBuild should be paired. The msbuild property writes that value to runtimeconfig.json. The ENV is for more flexible configuration on (typically) a process basis.